### ISSUE TITLE
Satellite - Mod_gearman problem

### ISSUE THREAD

**Issue by kbilev** [it-novum/openITCOCKPIT#1682]
**Describe the bug**
On Satellite systems with version 4.8.0 (RHEL and Ubuntu) the keyfile `/opt/openitc/etc/mod_gearman/secret.file` is missing.
In the log file `/opt/openitc/logs/mod_gearman/mod_gearman_worker.log` we see that the mod-gearman-worker-go is doing a panic and restarts all the time.
```
[Error][logger.go:72] *************************
[Info][mod_gearman_worker.go:134] mod-gearman-worker-go shutdown complete
[Error][logger.go:68] ********* PANIC *********
[Error][logger.go:69] Panic: could not open keyfile
[Error][logger.go:70] **** Stack:
[Error][logger.go:71] goroutine 1 [running]:
```

**Versions**
* openITCOCKPIT Server Version: 4.8.0
*  Ubuntu and RHEL

**Additional context**
mod-gearman-worker-go starts successfull after:
```
mkdir /opt/openitc/etc/mod_gearman
chmod o+rx /opt/openitc/etc/mod_gearman
echo "0c013ab34d3de983fd12e69f2895995b7" > /opt/openitc/etc/mod_gearman/secret.file
chown nagios:nagios /opt/openitc/etc/mod_gearman/secret.file
chmod u-w /opt/openitc/etc/mod_gearman/secret.file
chmod go-r /opt/openitc/etc/mod_gearman/secret.file
```

**Comment by kbilev** [comment]
Hi,
The workaround has a missing step. the `/opt/openitc/etc/mod_gearman/` folder must be created before the secret.file is created.
But I think that there is another problem with the mod-gearman-workers on the satellites.
maybe we could check via Discord or Teams?

**Comment by nook24** [comment]
This should be fixed with the release of openITCOCKPIT 4.8.1. Please reopen if the issue is still around.
https://openitcockpit.io/blog/posts/2024/2024-08-27-openitcockpit-4.8.1/
